src/filterScreenings.js

```diff
@@ -0,0 +1,71 @@
+/*Filters out ten screenings in the next five days or removes the last day's screenings incase
+the number of screenings pushes the total number of screenings to more than ten*/
+export function filterScreenings(data) {
+  let filteredScreenings = [];
+  data.forEach((screening) => {
+    
+    // Setting a starting data for the next day to start displaying screenings
+    const currentScreeningDate = new Date(screening.attributes.start_time);
+    const startingPoint = new Date(Date.now() + 86400000);
+    startingPoint.setHours(0);
+    startingPoint.setMinutes(0);
+    startingPoint.setSeconds(0);
+    startingPoint.setMilliseconds(0);
+
+    //Setting an end date for the screenings to be displayed
+    const endPoint = new Date(Date.now() + 518400000);
+    endPoint.setHours(0);
+    endPoint.setMinutes(0);
+    endPoint.setSeconds(0);
+    endPoint.setMilliseconds(0);
+
```

Comment:
This could be simplified by using startingPoint variable:

```javascript
const endPoint = new Date();
endPoint.setDate(startingPoint.getDate()+7);
endPoint.setHours(0,0,0,0);
```

Having to calculate how far in the future "518400000" is supposed to be is quite confusing 😁

Reply:
Haha yeah that's true, I will make some changes